### ISSUE TITLE
Everything I downloaded vanished ????? 

### ISSUE THREAD

**Issue by MalcolmInTheMiddle** [johanneszab/TumblThree#373]
Hi I'm new to this so, please bear with me. 

Well, I needed to backup my blog so I downloaded tumblthree, when I clicked exe. in the folder it opened up, so far so good.  I added my url and started the download process, after a week of downloading it was almost complete images and video files were in the folder I created on my pen drive. 

All of a sudden all the downloaded files vanished with out a trace, the same amount of space was being occupied on the drive? It wasn't localized either this event then manage to cause other new files unrelated to tumblthree to also vanished. In a panic I downloaded tumblr ripper which finished at 100% all the downloaded image and video files were in the folder on the pen drive I assigned them to.
I ejected the drive and plugged it back in and a now even all the files from tumblr ripper had vanished.

I ran check disk procedures and ended up with a folder full of CHK files at first I tried to restore them but the files were in a folder that could not be unhidden so could run the restore program. Eventually when I was able to get the chk  restore the program to work I ended up with a folder of unplayable MP4 files ??????

I thought this must be this drive maybe it has gone corrupt for whatever reason shrugs.
So I then decided in a panic to use a SD card in instead and retry downloading my blog on to it in an assigned folder, it started and files were appearing the folder when I awakened in the morning the program had encountered and issue with my internet ok no prob I couldn't get to start back up so I was forced to close the program. So I ejected the SD card out of curiosity and then put it back in, now even the few files tumblthree had downloaded had also vanished again. I ran check disk and the same corruption had occurred again ??????? whats happening ??????. I am running tumblr ripper as I speak  I'm afraid that once it is done and all the file have been downloaded that they too will disappear.

**Comment by Taranchuk** [comment]
I am sure that this is something related to your computer and not to the program. This may be a virus or technical problem. For now, as an option, try to download only the URL list, then upload it somewhere on the Internet and download all the files from the URL list on another computer (using wget or other tool).

**Comment by MalcolmInTheMiddle** [comment]
Thank you I will try this

**Comment by MalcolmInTheMiddle** [comment]
Nope I ran malwarebytes and AVG virus scan nothing found. Tumblthree would get to the end of the download then stop and the folder disappears same space occupied though

**Comment by johanneszab** [comment]
To be honest, that you end up having CHK files sounds more like an issue with the file system, or possibly a related hardware issue of your disk. 

Also, I've never heard of this before connected to TumblThree usage. I've also tested several times what happens if there is no space left on the device. In this cases, TumblThree will display an error message and stop crawling/downloading, but not randomly delete files.

>  In a panic I downloaded tumblr ripper which finished at 100% all the downloaded image and video files were in the folder on the pen drive I assigned them to.
I ejected the drive and plugged it back in and a now even all the files from tumblr ripper had vanished.

If I were you, I'd `chkdsk /F DEVICELETTER:` your drive, or use the Explorer/Computer Management to do so. And eventually check the S.M.A.R.T. values of that disk.
